golem_messages/message/tasks.py

```diff
@@ -634,3 +634,28 @@ class REASON(datastructures.StringEnum):
     @base.verify_slot('cannot_compute_task', CannotComputeTask)
     def deserialize_slot(self, key, value):
         return super().deserialize_slot(key, value)
+
+@library.register(TASK_MSG_BASE + 31)
+class StateUpdate(base.Message):
+
```

Comment:
@inexxt please add some docstring about the expected usage of this message

Reply:
Fixed